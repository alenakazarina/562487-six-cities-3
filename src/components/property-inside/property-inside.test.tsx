import * as React from 'react';
import * as renderer from 'react-test-renderer';
import PropertyInside from './property-inside';

const AMENITIES = [
  `Kitchen`, `Heating`, `Washing machine`
];

describe(`PropertyInside`, () => {
  it(`should render PropertyInside correctly`, () => {
    const tree = renderer.create(
        <PropertyInside amenities={AMENITIES} />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
