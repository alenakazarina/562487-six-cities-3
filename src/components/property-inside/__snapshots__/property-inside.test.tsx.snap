// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PropertyInside should render PropertyInside correctly 1`] = `
<div
  className="property__inside"
>
  <h2
    className="property__inside-title"
  >
    What's inside
  </h2>
  <ul
    className="property__inside-list"
  >
    <li
      className="property__inside-item"
    >
      Kitchen
    </li>
    <li
      className="property__inside-item"
    >
      Heating
    </li>
    <li
      className="property__inside-item"
    >
      Washing machine
    </li>
  </ul>
</div>
`;
